### PR TITLE
Proceed to Checkout Button Link: Add code snippet and screenshot

### LINKED ISSUE #10914: `Proceed to Checkout Button Link`: Add code snippet and screenshot
Recently, we've updated the "Available Extensibility Interfaces for the Cart and Checkout blocks" post. In this post, we introduced new code snippets and screenshots. This issue aims to add the following code snippet and screenshot to the [`Proceed to Checkout Button Link`](https://github.com/woocommerce/woocommerce-blocks/blob/6083484477410fd99d1efd5ca909288eb4d19963/docs/third-party-developers/extensibility/checkout-block/available-filters.md?plain=1#L107) section:

```js
const { registerCheckoutFilters } = window.wc.blocksCheckout;
 
// Adjust the proceed to checkout button link.
registerCheckoutFilters( 'example-extension', {
  proceedToCheckoutButtonLink: ( value, extensions, args ) => {
    if ( ! args?.cart.items ) {
      return value;
    }
 
    const isSunglassesInCart = args?.cart.items.some(
      ( item ) => item.name === 'Sunglasses'
    );
 
    // Return the default value if sunglasses is not in the cart.
    if ( ! isSunglassesInCart ) {
      return value;
    }
 
    return '/sunglasses-checkout';
  }
} );
```

screenshot-2023-08-24-at-13 39 35

### PR DESCRIPTION
**Pull request by masteradhoc**
## What
Add code snippet and screenshot for the Proceed to Checkout Button Link Section
Fixes #10914 

## Why
Complete the documentation for the Proceed to Checkout Button Link Section

## Testing Instructions
A reviewer should look up the changed *.md file and ensure that the added code and the screenshot match with the ones mentioned in the ticket

* [ ] Do not include in the Testing Notes 
* [ ] Should be tested by the development team exclusively 

## WooCommerce Visibility
Required:

* [ ] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental
* [x] N/A

## Checklist

Required:
* [x] This PR has either a `[type]` label or a `[skip-changelog]` label.
* [ ] This PR is assigned to a milestone.

Conditional:
* [ ] This PR has a changelog description (if `[skip-changelog]` label is not present).
* [ ] This PR adds/removes a feature flag & I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
* [ ] This PR adds/removes an experimental interfaces, and I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
* [ ] This PR has been accessibility tested.
* [ ] This PR has had any necessary documentation added/updated.

## Changelog

> Add suggested changelog entry here.
Update code snippet and screenshots for the documentation

### CODE REVIEW

File: docs/third-party-developers/extensibility/checkout-block/available-filters.md [hunk 1]
```diff
@@ -151,6 +151,31 @@ The Cart block contains a button which is labelled 'Proceed to Checkout' and lin
...
+    // Return the default value if sunglasses is not in the cart.
+    if ( ! isSunglassesInCart ) {
+      return value;
+    }
+ 
+    return '/sunglasses-checkout';
+  }
+} );
+```
+<img width="761" alt="image" src="https://user-images.githubusercontent.com/3323310/267270560-2964d529-771d-4d00-9fff-646b15d59cbf.png">
```
Comment (nielslange) [inline docs/third-party-developers/extensibility/checkout-block/available-filters.md]:
````suggestion
```

<img width="761" alt="image" src="https://user-images.githubusercontent.com/3323310/267270560-2964d529-771d-4d00-9fff-646b15d59cbf.png">
````

**Review by nielslange — CHANGES_REQUESTED** [review]
Thanks for working on this, @masteradhoc. I noticed that there's no line break below the code fence block, which would lead to an MD linting error. I suggested a change in my review.

**Comment by masteradhoc** [comment]
@nielslange fixed, thanks for the heads up.

**Review by nielslange — APPROVED** [review]
Thank you so much for working on this, @masteradhoc. Yet again, you made the docs just a little bit better! 🙌

**Pull request merged**